src/notebook_utils/notebook_utils.py

```diff
@@ -0,0 +1,41 @@
+import itables.options as opt
+from itables import init_notebook_mode, show
+from pandas import DataFrame
+
+
+def display_df(
+        df: DataFrame,
+        layout: dict = None,
+        buttons: list = None,
+        length_menu: list = None
+) -> None:
+    """
+    Display a pandas DataFrame using itables.
+    iTables project page: https://github.com/mwouts/itables
+
+    :param df: a pandas DataFrame
+    :param layout: layout options, refer to https://datatables.net/reference/option/layout
+    :param buttons: buttons options, options refer to https://datatables.net/reference/button/
+    :param length_menu: length menu options
+    :return:
+    """
+    # initialize itables for the notebook
+    init_notebook_mode(all_interactive=False)
```

Comment:
Is this ok to run many times? Seems like it's something you'd run once

Reply:
Ok, but is it ok to do that? I don't know if there are any repercussions to running what looks like a one time function many times

Reply:
I think it's probably ok to do that. Seems like the func is setting pandas df styler and _repr_html_. If we just do it once at the beginning, it can be overwritten I think. Then I don't know the behavior of the below code.  
https://github.com/mwouts/itables/blob/main/src/itables/javascript.py#L75

Reply:
:+1: